bentoml/adapters/dataframe_input.py

```diff
@@ -55,6 +58,92 @@ class DataframeInput(BaseInputAdapter):
         ValueError: Incoming data format can not be handled. Only json and csv
     """
 
+    def from_cli(self, cli_args: Tuple[str, ...]) -> Iterator[InferenceTask]:
+        parser = argparse.ArgumentParser()
```

Comment:
Why not call `parse_cli_input` here?

Reply:
I don't think I can add the `json` and `csv` options then, right?

Reply:
We could get the file ext ".csv" or ".json" like this:

for bin_io in parse_cli_input(cli_args):
    file_ext = bin_io.name

Reply:
Oh ok, what would I do for the string input?